.github/workflows/main.yaml

```diff
@@ -47,6 +47,12 @@ jobs:
         run: make lint
         working-directory: ./src/github.com/${{ github.repository }}
 
+      - name: Run link check
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'yes'
+          use-verbose-mode: 'yes'
```

Comment:
@Rei1010 Thanks for the contributions, and could you please add more description for these flags?

The action will only check the files in PR or all the files in repo include vendor?  

https://github.com/netdata/netdata/blob/master/.github/workflows/docs.yml#L28 For this project, they also have some other flags and the config file. We need to have some research.

/cc @gy95